cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsLaunchService.java

```diff
@@ -164,26 +163,6 @@
         return awsResourceConnector.check(ac, instances);
     }
 
-    private ListStackResourcesResult updateCloudFormationStack(AuthenticatedContext ac, CloudStack stack, ModelContext modelContext) {
```

Comment:
So removing this was necessary to persist the updated stack during load balancer creation? I'm sure it was but I can't tell why you removed it nor if it was doing anything important initially.

Reply:
Ah, no, this is unrelated. It just so happened when I was testing this code I realized this method wasn't being used anymore, and decided to clean it up.